libs/features/personalization/personalization.js

```diff
@@ -559,13 +561,28 @@ export async function applyPers(manifests) {
   const cleanedManifests = cleanManifestList(manifests);
 
   let results = [];
+  const experiments = [];
   for (const manifest of cleanedManifests) {
-    results.push(await runPersonalization(manifest, config));
+    const noOverride = !config.mep?.override;
```

Comment:
wouldn't it be clearer to have an override flag, and then !override test?

Reply:
fixed